axon-cosmosdb/src/test/java/org/axonframework/extensions/cosmosdb/eventsourcing/tokenstore/CosmosTokenItemTest.java

```diff
@@ -55,14 +59,17 @@ void initialTokenCreationWorks() {
     }
 
     @Test
-    void releaseWorksAndUpdatesTheTimeStamp() throws InterruptedException {
+    void releaseWorksAndUpdatesTheTimeStamp() {
```

Comment:
Is this fix per chance accidentally added to this branch, whereas it belongs to the branch for PR #1?

Reply:
Something like that. But let's say it's boyscouting from the sonar issues.